src/test/groovy/hudson/plugins/gradle/injection/BuildScanInjectionMavenIntegrationTest.groovy

```diff
@@ -583,6 +588,39 @@ node {
         "+:this-one-does-not-match\n+:this-one-too"     | "withMaven(maven: 'mavenInstallationName') {"                         | false
     }
 
+    def 'vcs repository pattern injection for freestyle remote project - #filter #shouldApplyAutoInjection'(String filter, boolean shouldApplyAutoInjection) {
+        given:
+        mavenInstallationRule.mavenVersion = '3.9.2'
+        mavenInstallationRule.addInstallation()
+        withInjectionConfig {
+            vcsRepositoryFilter = filter
+        }
+        createSlaveAndTurnOnInjection()
+
+        def p = j.createFreeStyleProject()
+        p.buildWrappersList.add(new BuildScanBuildWrapper())
```

Comment:
This wrapper is not needed for the test that we are doing. Might only be needed for UI tests.

Reply:
Yup, removed